prog/mutation.go

```diff
@@ -349,6 +349,8 @@ func (t *BufferType) mutate(r *randGen, s *state, arg Arg, ctx ArgCtx) (calls []
 		}
 	case BufferFilename:
 		a.data = []byte(r.filename(s, t))
+	case BufferGlob:
+		a.data = r.randString(s, t)
```

Comment:
There is one interesting case here: t.Values can be empty, then randString will produce a random string. But this is a file name and a random string can/will break test sandboxing (test programs will start operating on random global files).
If t.Values is empty we could disable the syscall in UpdateGlobFilesForType, or we can do something like this here:
```
if len(t.Values) != 0) {
    a.data = r.randString(s, t)
else {
    a.data = []byte(r.filename(s, t))
}
```